mokapot/model.py

```diff
@@ -290,8 +315,8 @@ def fit(self, psms):
             norm_feat = norm_feat[shuffled_idx, :]
             start_labels = start_labels[shuffled_idx]
 
-        # Prepare the model:
-        model = _find_hyperparameters(self, norm_feat, start_labels)
+        if not self.supervised_step:
+            model = _find_hyperparameters(self, norm_feat, start_labels)
```

Comment:
Remove this. We still need to find the optimal hyperparameters for the semi-supervised phase of model training.